server/statistics/store.go

```diff
@@ -69,12 +79,28 @@ func (s *StoresStats) Observe(storeID uint64, stats *pdpb.StoreStats) {
 	rollingStoreStat.Observe(stats)
 }
 
+// ObserveRegionStats records the current stores status from region stats.
+func (s *StoresStats) ObserveRegionStats(storeIDs []uint64, writeBytesRates, writeKeysRates []float64) {
```

Comment:
Have you compared the data before and after smoothing? I think the statistics of the region itself are relatively smoother because the reporting period is much longer.

Reply:
Region reporting interval is not stable